lib/excon/connection.rb

```diff
@@ -46,6 +46,7 @@ def proxy=(new_proxy)
     #     @option params [String] :ciphers Only use the specified SSL/TLS cipher suites; use OpenSSL cipher spec format e.g. 'HIGH:!aNULL:!3DES' or 'AES256-SHA:DES-CBC3-SHA'
     #     @option params [String] :proxy Proxy server; e.g. 'http://myproxy.com:8888'
     #     @option params [Fixnum] :retry_limit Set how many times we'll retry a failed request.  (Default 4)
+    #     @option params [Fixnum] :retry_interval Set how long to wait between retries.
```

Comment:
Maybe we could just leave your stuff as-is, and later add another/distinct option, so you could do `connection.request(:idempotent => true, :retry_limit => 6, :exponential_backoff => true)` or something. That might be easiest, rather than trying to more generically do it. What do you think?